tests/test_jenkins.py

```diff
@@ -161,6 +162,24 @@ def test_fetch_empty(self):
 
         self.assertEqual(len(builds), 0)
 
+    @httpretty.activate
```

Comment:
For this test you can also add a check to test if that item was not retrieved. For instance, you can use `assertNotIn()` method with `builds` list.